### PR TITLE
Added footer to app. Also Added logo svg for navbar.

### PR DESCRIPTION
**Pull request by Sosobo-code**

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
+import Footer from "@/components/Footer";
 
 export default function Home() {
   return (
-    <HomePage/>
+      <>
+        <HomePage/>
+        <Footer />
+      </>
+
```
Comment (Crystallized21) [inline src/app/page.tsx]: Remove this unnecessary space

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Footer = () => {
+    return (
+        <div className="text-center p-3 bg-primary text-white">
```
Comment (Crystallized21) [inline src/components/Footer.tsx]: Change `bg-primary` to `bg-background` so shadcn can recognise light and dark mode

File: src/components/HomePage.tsx [hunk 3]
```diff
@@ -19,6 +19,7 @@ const HomePage = () => {
                 <Button className="w-full md:w-1/2 sm:w-1/8">Get Started</Button>
             </div>
         </div>
+
```
Comment (Crystallized21) [inline src/components/HomePage.tsx]: ???

File: src/components/NavBar.tsx [hunk 4]
```diff
@@ -36,21 +37,9 @@ export default function NavBar() {
...
-              strokeLinecap="round"
-              strokeLinejoin="round"
-              className="h-6 w-6"
-            >
-              <circle cx="12" cy="12" r="10"/>
-              <path d="M12 16v-4"/>
-              <path d="M12 8h.01"/>
-            </svg>
-            <span className="font-bold">Logo</span>
+
```
Comment (Crystallized21) [inline src/components/NavBar.tsx]: Unnecessary empty lines

**Review by Crystallized21 — CHANGES_REQUESTED** [review]

File: src/components/NavBar.tsx (hunk 4 above)
Comment (Sosobo-code) [reply src/components/NavBar.tsx]: mb

File: src/components/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Footer = () => {
+  return (
+    <div className="text-center p-3 bg-primary text-white">
```
Comment (Crystallized21) [inline src/components/Footer.tsx]: `bg-background` to match shadcn themes

**Review by Crystallized21 — CHANGES_REQUESTED** [review]
bruh

**Review by Crystallized21 — APPROVED** [review]
lgtm

**Pull request merged**